packages/create-ripple-app/template/_tests/_smoke/test/e2e/integration/smoke/DemoLandingPage.feature

```diff
@@ -22,7 +22,7 @@ Feature: Demo Page
     And the body markup component should exist
     And the featured news list component should exist
     And the featured card navigation component should exist
-    And the card event component should exist
+    # And the card event component should exist
```

Comment:
I will say not add this demo landing page test for the custom site.

1. In content vic, only vic gov au has that page, none of other sites has it in backend.
2. In any other independent site, only the main site will have this page. So most of the time I think this tests won't work.

Reply:
yeah i agree - need to write docs on how to add / remove tests to sites first 😞